build/rust/Cargo.toml

```diff
@@ -7,6 +7,9 @@ edition = "2018"
 adblock-ffi = { path = "../../vendor/adblock_rust_ffi" }
 challenge-bypass-ristretto-ffi = { path = "../../vendor/challenge_bypass_ristretto_ffi" }
 speedreader-ffi = { path = "../../components/speedreader/rust/ffi" }
+[target."cfg(target_os = \"windows\")".dependencies]
```

Comment:
this seems fragile for the same reason as the gn configs https://github.com/brave/brave-core/pull/7365/files#r538591095
can we pass an equivalent to `enable_rpill` that is set in gn?

Reply:
Agree, would set up an optional dependency that's enabled as a feature

Reply:
Thanks for the suggestion! I'm resolving this thread as we're not relying on Rust anymore